documentation/asciidoc/accessories/touch-display-2/about.adoc

```diff
@@ -127,3 +127,23 @@ To specify these options, add them, separated by commas, to your `dtoverlay` lin
 ----
 dtoverlay=vc4-kms-dsi-ili9881-7inch,sizex=400,invx,invy
 ----
+
+=== Installation and software setup on Compute Module based devices.
+
+All Raspberry Pi SBCs auto-detect the official touch displays as the wiring to them is known, and this autodetection ensures the correct Device Tree entries are passed to the kernel. However, Compute Modules are intended for industrial applications where the integrator is entitled to use any and all GPIOs and interfaces for whatever purposes they require. Autodetection is therefore not feasible, and hence is disabled on Compute Module devices. This means that the Device Tree fragments required to set up the display need to be loaded via some other mechanism, which can be either with a dtoverlay entry in config.txt or via a custom base DT file.
```

Comment:
"wiring to them is known" -> "circuitry connected to the DSI connector is fixed" ?
"feasible" -> "possible" ?

"which can be either with a dtoverlay entry in config.txt or via a custom base DT file." - could also be a dtoverlay in the HAT eeprom? (but perhaps this is so specialised that it's not worth mentioning)